packages/node-cli/src/loggers/NodeConsoleLogger.js

```diff
@@ -29,7 +29,7 @@ class NodeConsoleLogger extends ConsoleLogger {
 		this.println(message, COLORS.ERROR);
 	}
 
-	_formatString(str, options) {
+	formatString(str, options) {
```

Comment:
Why removing the underscore? I would keep it because is a kind of "private" function. If you agree I would not modify this file at all.

Reply:
We can follow the solution we talk about yesterday:

- Do the **bold** and **important** color (red) in the important(message) method and leave this function with the underscore
- Use one single message with the important(message) (composed by IMPORTANT and LICENCSE_INTRO) from the Licencse.js -> display 
- Add the important(message) to the ConsoleLogger interface

Reply:
Changes look good.